docs/admin/cluster-management.md

```diff
@@ -136,7 +136,7 @@ The initial values of the autoscaler parameters set by ``kube-up.sh`` and some m
 or using gcloud CLI:
 
 ```
-gcloud preview autoscaler --zone compute-zone <command>
+gcloud alpha compute autoscaler --zone $ZONE <command>
```

Comment:
apparently this is the only file that still refers to ```compute-zone```, and all others use ```--zone $ZONE``` so its good to clean these out of the k8s library